cmake/modules/AddRootBench.cmake

```diff
@@ -50,6 +50,36 @@ function(RB_ADD_GBENCHMARK benchmark)
                                               TIMEOUT "${TIMEOUT_VALUE}" LABELS "${ARG_LABEL}" RUN_SERIAL TRUE)
 endfunction(RB_ADD_GBENCHMARK)
 
+
+#----------------------------------------------------------------------------
+# function RB_ADD_PYTESTBENCHMARK(file_name)
+#
+# Dependency: pytest with pytest-benchmark and PyROOT modules
+#
+#----------------------------------------------------------------------------
+function(RB_ADD_PYTESTBENCHMARK file_name)
+  if(ROOT_pyroot_experimental_FOUND)
+    set(ROOT_ENV ROOTSYS=${ROOTSYS}
+        PATH=${ROOTSYS}/bin:$ENV{PATH}
+        LD_LIBRARY_PATH=${ROOTSYS}/lib:${ROOTSYS}/lib/${python_dir}:$ENV{LD_LIBRARY_PATH}
+        PYTHONPATH=${ROOTSYS}/lib:${ROOTSYS}/lib/${python_dir}:$ENV{PYTHONPATH})
```

Comment:
Can it be that this `${python_dir}` here is a remnant of when we had python subdirectories? It seems to me that the environment for PyROOT experimental and old PyROOT should be set in the same way.

Reply:
I need to check this with Massimiliano, looks like I actually don't need this anymore...